Erreur inconnue (veuillez faire un rapport de bug) : <type 'exceptions.AttributeError'>\n
Sa valeur est : AttributeError("'tuple' object has no attribute 'start'",)
  File "<script>", line 43, in <module>
  File "<script>", line 20, in init
